src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V1/DownloadedPackagesStoreV1.swift

```diff
@@ -18,15 +18,33 @@
 import Foundation
 import FMDB
 
+enum SQLiteError: Int32 {
+	case sqlite_full = 13
+	case unknown = -1
+}
```

Comment:
Can you please add documentation where to find these values.

Reply:
Also, is `Int32` needed? Name and type of this enum is ambiguous. What about:

```suggestion
enum SQLiteErrorCode: Int32 {
  	case sqlite_full = 13
  	case unknown = -1
}
```
(please check white spaces as this web-foo adds some)

Reply:
or…

```suggestion
enum SQLiteError: Error, Int32 {
  	case sqlite_full = 13
  	case unknown = -1
}
```

Reply:
👍 
Renamed the enum name to SQLiteErrorCode